e3db-crypto-android/src/main/java/com/tozny/e3db/crypto/SecureStringManager.java

```diff
@@ -0,0 +1,76 @@
+package com.tozny.e3db.crypto;
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * 
+ * Copyright (c) 2018 
+ * 
+ * All rights reserved.
+ * 
+ * e3db-java
+ * 
+ * Created by Lilli Szafranski on 4/6/18.
+ * 
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+
+
+import android.content.Context;
+import org.jetbrains.annotations.NotNull;
+
+import javax.crypto.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+
+class SecureStringManager {
+
+    private static void checkArgs(Context context, String fileName, String string) throws Exception { // TODO: Lilli, move higher
+        if (context == null) {
```

Comment:
Consider using (and/or extending) the `Checks` class which has methods for this.